### ISSUE TITLE
Feature Request: NeoDash Helm Chart for Kubernetes based deployments

### ISSUE THREAD

**Issue by venkatalolla** [neo4j-labs/neodash#884]
Hi,

**Is your feature request related to a problem? Please describe.**
This is a new feature request to create a Helm chart to support the Kubernetes based deployments in a package pattern. This Helm chart should enable the NeoDash deployment in either editor or reader mode for easy versioning, packaging and installing these dashboards at an enterprise level.

**Describe the solution you'd like**
The solution includes a Helm chart with the following Kubernetes resources templates to support the editor or reader mode for ease of deployment,

- Deployment
- Service
- Service Account
- Ingress (Optional)
- Values file (Default configurations)

Along with the Helm chart, add the appropriate documentation about the usage and default values.

**Describe alternatives you've considered**
The only alternative is to use the Kubernetes YAML files to deploy the NeoDash.

**Additional context**
I have successfully deployed the NeoDash version v2.4 and would like to contribute this base Helm chart to the community with optional ingress feature for the extending it to use with an existing ingress controller setup.

Cheers,
Surya Lolla

**Comment by adriens** [comment]
Yup ,including configuration ointing to where the json export of the dashboard ould be awesome :pray:

**Comment by venkatalolla** [comment]
> Yup ,including configuration ointing to where the json export of the dashboard ould be awesome :pray: 

Sure, are you referring to the dashboard json once published to the Neo4j database?
